milestone2.ts

```diff
@@ -164,7 +178,10 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) => {
+  let person =  data.filter((vol: Volunteer) => vol.age > 40).find((vol: Volunteer) => vol.city === "SLO")
```

Comment:
You can actually use two conditions in the same call to find if you want to cut the filter out here. You would combine with the **and** operator: **&&** . For example, find(vol => vol.age > 40 && vol.city === "SLO"). This will make your code much more readable and cut down your function calls!